defaults/main.yml

```diff
@@ -4,7 +4,9 @@ zookeeper_playbook_version: "0.9.2"
 zookeeper_version: 3.4.6
 zookeeper_url: http://www.us.apache.org/dist/zookeeper/zookeeper-{{zookeeper_version}}/zookeeper-{{zookeeper_version}}.tar.gz
 
+zookeeper_debian_apt_install: false
```

Comment:
I would suggest to prefer apt but since 3.4.6 is not available and released - lets leave priority to tarball install